### DISCUSSION TITLE
Error Creating simulation on PK-Sim 10.0

### DISCUSSION THREAD

**Discussion by ooladoyinbo** [Open-Systems-Pharmacology/Forum#1005] [Models] [answered]
Hi PK-Sim Community,

I am a PBPK beginner and I need your help. I downloaded a model from OSP that was initially created on PK-sim 9.1. However, opening this project in PK-sim 10.0 and trying to create a simulation for a population showed this error: 

![image](https://user-images.githubusercontent.com/92388188/139317976-4748d5e7-80a9-4cf8-9cce-6fe29f6b99f5.png)

The only difference in my population model versus the template model was the variability I included in the transporter values for the drug. Could it be a bug in the new PK-sim version OR did I set up my simulation wrongly?

Your help is appreciated in advance.

Thanks
Olayide

**Comment by Yuri05** [comment, marked as answer]
It's a bug. https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1929
